### PR TITLE
test(integration): add more tests

### PR DESCRIPTION
**Pull request by ikatyang**
Context: https://github.com/prettier/prettier/pull/2730#issuecomment-326788006

Additionally, I accidentally put `globby.sync()` outside of `try..catch` block, put them back now. 😅

### CODE REVIEW

File: tests_integration/cli/write/invalid.js [hunk 1]
```diff
@@ -0,0 +1 @@
+this is invalid!
```
Comment (azz) [inline tests_integration/cli/write/invalid.js]: This file needs to be ignored from eslint

Comment (ikatyang) [reply tests_integration/cli/write/invalid.js]: Oops, forgot to turn off this one.

Comment (lydell) [reply tests_integration/cli/write/invalid.js]: We should be able to add `/tests_integration/cli/` to .eslintignore and get rid of all `eslint-disable` comments in `tests_integration/cli/`.

Comment (lydell) [reply tests_integration/cli/write/invalid.js]: @ikatyang Are you going to remove those `eslint-disable` comments, or do you want me to? :)

Comment (ikatyang) [reply tests_integration/cli/write/invalid.js]: I'm removing them. 😎

**Pull request merged**